From e8aba58ab19a18f83d7f78e80d5e4f51e7e4e8a9 Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Fri, 25 Oct 2019 12:20:22 -0400
Subject: [PATCH] add missing case mapping between U+03F3 and U+037F

somehow this seems to have been overlooked. add it now so that
subsequent overhaul of case mapping implementation will not introduce
a functional change at the same time.
---
 src/ctype/towctrans.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ctype/towctrans.c b/src/ctype/towctrans.c
index 8f681018..9b91b2de 100644
--- a/src/ctype/towctrans.c
+++ b/src/ctype/towctrans.c
@@ -176,6 +176,7 @@ static const unsigned short pairs[][2] = {
 	{ 0x243, 0x180 },
 	{ 0x244, 0x289 },
 	{ 0x245, 0x28c },
+	{ 0x37f, 0x3f3 },
 	{ 0x3f4, 0x3b8 },
 	{ 0x3f9, 0x3f2 },
 	{ 0x3fd, 0x37b },
-- 
2.24.1

